ios/src/ImageCropPicker.m

```diff
@@ -36,17 +49,32 @@
 #define ERROR_CANNOT_PROCESS_VIDEO_KEY @"E_CANNOT_PROCESS_VIDEO"
 #define ERROR_CANNOT_PROCESS_VIDEO_MSG @"Cannot process video data"
 
+#import "UIImage+Resize.h"
+#import "UIImage+Extension.h"
+#import "Compression.h"
+
 @implementation ImageResult
 @end
 
+@interface ImageCropPicker(UIImagePickerControllerDelegate) <UIImagePickerControllerDelegate>
```

Comment:
different categories for each protocol conformance to make simplify navigating the code